### PR TITLE
LPS-108557 Move draggable and selectable props to display context

### PR DESCRIPTION
**Pull request by carloslancha**

### CODE REVIEW

**Comment by carloslancha** [comment]
ci:test:sf

File: modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/js/miller_columns/MillerColumnsColumnItem.es.js [hunk 1]
```diff
@@ -57,51 +51,35 @@ const MillerColumnsColumnItem = ({
...
-		[actionHandlers]
-	);
+		actions.map(action => {
+			if (!action.quickAction && action.url) {
+				dropdownActions.push({
+					...action,
+					handler: action.handler || actionHandlers[action.id] || noop
+				});
+			}
+		});
```
Comment (wincent) [inline modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/js/miller_columns/MillerColumnsColumnItem.es.js]: Not `map`, `forEach` instead.

File: modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/js/miller_columns/MillerColumnsColumnItem.es.js [hunk 2]
```diff
@@ -57,51 +51,35 @@ const MillerColumnsColumnItem = ({
...
-	const dropdownActions = useRef(getDropdownActions(actions));
-	const quickActions = useRef(getQuickActions(actions));
+		actions.map(action => {
+			if (action.quickAction && action.url) {
+				quickActions.push({
+					...action,
+					handler: action.handler || actionHandlers[action.id] || noop
+				});
+			}
+		});
```
Comment (wincent) [inline modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/js/miller_columns/MillerColumnsColumnItem.es.js]: Same here: `map` → `forEach`

**Comment by carloslancha** [comment]
Sent to https://github.com/ealonso/liferay-portal/pull/2781

**Pull request closed without merging**